Microsoft.HealthVault/Clients/Deserializers/ThingDeserializer.cs

```diff
@@ -144,5 +189,87 @@ public ThingDeserializer(IHealthVaultConnection connection)
 
             return result;
         }
+
+        public ThingBase DeserializeItem(XPathNavigator thingNav)
+        {
+            ThingBase result;
+            Guid typeId = new Guid(thingNav.SelectSingleNode("type-id").Value);
```

Comment:
Then we should mark it as internal right?

---
In reply to: [123105225](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/62#discussion_r123105225) [](ancestors = 123105225,123101596)